.github/workflows/generate-binaries.yaml

```diff
@@ -44,14 +44,18 @@ jobs:
           echo "$PWD" >> $GITHUB_PATH
 
       - name: Build binaries
-        run: cargo build --features=ispc
+        run: |
+          cargo build --features=ispc --target=x86_64-apple-darwin
+          cargo build --features=ispc --target=aarch64-apple-darwin
 
       - uses: actions/upload-artifact@v3
         with:
           name: ISPC kernels
           path: |
             src/ispc/libkernel{,_astc}x86_64-apple-darwin.a
             src/ispc/libispc_texcomp_astcx86_64-apple-darwin.a
+            src/ispc/libkernel{,_astc}aarch64-apple-darwin.a
+            src/ispc/libispc_texcomp_astcaarch64-apple-darwin.a
```

Comment:
Fwiw I had some trouble (https://github.com/Traverse-Research/intel-tex-rs-2/commit/fa09a41929b3484772d14e6a634b7955c77601eb)
getting `{}` in globs to work and https://github.com/actions/toolkit/tree/main/packages/glob#patterns doesn't state support for this, we'll have to write them out :(